AEPMessaging/Sources/LiveActivity/AEPLiveActivityAttributes.swift

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
AEPLiveActivityAttributes & AEPLiveActivityData class should be available to customer's both the Main App and WidgetExtension.

In future planning to keep this code in a separate library